cpp/CMakeLists.txt

```diff
@@ -49,7 +84,7 @@ function(tosca_add_compile_flags target)
     CXX_EXTENSIONS OFF)
   target_compile_options(${target} PRIVATE
     $<$<CXX_COMPILER_ID:MSVC>:/MP /W4 /external:W0>
-    $<$<CXX_COMPILER_ID:GNU,Clang,AppleClang>:-march=native -mtune=native -Wall -Wextra -Wpedantic -Wtype-limits -Wconversion -Wsign-conversion -Wdouble-promotion -g>
```

Comment:
why remove `-g` flag ?

Reply:
This kind of controversial, I will revert it.
The thing is that debug symbols were always enabled, even for release builds. This flag would be duplicated in some cases... 
I do not need debug symbols for release builds, and IMO if needed, should be optional. I will come back here to clean this flags some other day.